add docs on how to configure the FE and BE to serve SSL (#1793)
18F/e-QIP-prototype: docs/https.md

> # Instructions on running eapp over SSL in dev

Reviewer: ```suggestion
# Instructions on running eapp over SSL – for local development only
```
